contracts/consensus-gateway/ConsensusGateway.sol

```diff
@@ -205,4 +205,79 @@ contract ConsensusGateway is MasterCopyNonUpgradable, MessageBus, ConsensusGatew
             msg.sender
         );
     }
+
+    /**
+     * @notice Confirm withdraw in order to transfer most.
+     *
+     * @param _amount MOST token amount for withdrawal
+     * @param _beneficiary Address of beneficiary where tokens will be withdrawn.
+     * @param _feeGasPrice Gas price at which fee will be calculated.
+     * @param _feeGasLimit Gas limit at which fee will be capped.
+     * @param _blockNumber Block number of Aux chain against which storage
+                           proof is generated.
+     * @param _withdrawer Address of the withdrawer account.
+     * @param _rlpParentNodes Storage merkle proof to verify message declaration
+                              on the origin chain.
+     *
+     * @return messageHash_ Message Hash
+     */
```

Comment:
Please add `\pre` and `\post` conditions.

Reply:
Added.